system_tests/storage.py

```diff
@@ -31,9 +31,19 @@
 from retry import RetryResult
 
 
-retry_429 = RetryErrors(exceptions.TooManyRequests)
 HTTP = httplib2.Http()
-_helpers.PROJECT = TESTS_PROJECT
+
+
+def _bad_copy(bad_request):
+    """Predicate: pass only exceptions for a failed copyTo."""
+    err_msg = bad_request.message
+    return (err_msg.startswith('No file found in request. (POST') and
+            'copyTo' in err_msg)
```

Comment:
Sadly no. The entire payload of the response is `No file found in request.`, whereas most error responses contain a JSON payload with `errors`.